src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -535,10 +535,23 @@ public sealed partial class JsonStringEnumConverter : System.Text.Json.Serializa
         public override bool CanConvert(System.Type typeToConvert) { throw null; }
         public override System.Text.Json.Serialization.JsonConverter CreateConverter(System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options) { throw null; }
     }
-    public sealed partial class ReferenceHandling
+    public abstract partial class ReferenceHandler
+    {
+        protected ReferenceHandler() { }
+        public static System.Text.Json.Serialization.ReferenceHandler? Default { get { throw null; } }
```

Comment:
Because it holds a `null` value on the implementation.

Reply:
I removed the property since as you mentioned in another comment, it is no longer functional, this still might need approval since we are changing the API.